packages/calypso-products/src/plans-list.js

```diff
@@ -28,23 +31,24 @@ const getMonthlyTimeframe = () => ( {
 
 const getDotcomPlanDetails = () => ( {
 	// Features only available for annual plans
-	getAnnualPlansOnlyFeatures: () =>
-		compact( [
-			constants.FEATURE_FREE_DOMAIN,
-			constants.FEATURE_CUSTOM_DOMAIN,
-			constants.FEATURE_EMAIL_LIVE_CHAT_SUPPORT_ALL_DAYS,
-			constants.FEATURE_EMAIL_LIVE_CHAT_SUPPORT_BUSINESS_DAYS,
-			constants.FEATURE_LIVE_CHAT_SUPPORT,
-			constants.FEATURE_LIVE_CHAT_SUPPORT_BUSINESS_DAYS,
-			constants.FEATURE_LIVE_CHAT_SUPPORT_ALL_DAYS,
-		] ),
+	getAnnualPlansOnlyFeatures: () => [
+		constants.FEATURE_FREE_DOMAIN,
+		constants.FEATURE_CUSTOM_DOMAIN,
+		constants.FEATURE_EMAIL_LIVE_CHAT_SUPPORT_ALL_DAYS,
+		constants.FEATURE_EMAIL_LIVE_CHAT_SUPPORT_BUSINESS_DAYS,
+		constants.FEATURE_LIVE_CHAT_SUPPORT,
+		constants.FEATURE_LIVE_CHAT_SUPPORT_BUSINESS_DAYS,
+		constants.FEATURE_LIVE_CHAT_SUPPORT_ALL_DAYS,
+	],
 } );
 
+/* eslint-disable wpcalypso/jsx-classname-namespace */
 const plansDescriptionHeadingComponent = {
 	components: {
 		strong: <strong className="plans__features plan-features__targeted-description-heading" />,
 	},
 };
+/* eslint-enable */
```

Comment:
Did we mean to say `/* eslint-enable wpcalypso/jsx-classname-namespace */` instead? Otherwise, we'd enable any rules we've temporarily disabled, no?

Reply:
That's a good question. It was actually intentional, as per the recommendation on this internal post p6JqRr-3OA-p2#comment-10916 which I'll quote here:

> if disable is used with a reference to which rules are being disabled, don’t include those rules when adding the enable back in. Why? When someone sees the enable with the ruleset, they wouldn’t know if other rules have been disabled previously or not. When the enable doesn’t have the referenced rule(s), then the reader knows that every disable from earlier in the file, is now re-enabled.

But I don't feel strongly about this if you disagree.